pages/Documents/ConversationalAI/ConversationBuilder/Bots/bot-basics.md

```diff
@@ -22,16 +22,16 @@ There are two, general types of bots:
 
 ### Create a bot
 
-* For details on creating a custom bot, see [here](conversation-builder-bots-custom-bots.html).
-* For details on creating a post-conversation survey bot, see [here](conversation-builder-bots-post-conversation-survey-bots.html).
+* [Create a custom bot](conversation-builder-bots-custom-bots.html).
+* [Create a post-conversation survey bot](conversation-builder-bots-post-conversation-survey-bots.html).
 
 ### Configure bot settings
 
 {: .important}
-Post-conversation survey bots have a few, unique settings that custom bots don't have. For information on these settings, see [here](conversation-builder-bots-post-conversation-survey-bots.html#configure-the-survey-bots-settings).
+Post-conversation survey bots have [a few, unique settings](conversation-builder-bots-post-conversation-survey-bots.html#configure-the-survey-bots-settings) that custom bots don't have.
 
 **To configure bot settings**
-1. Open the bot, and click <img style="width:25px" src="img/ConvoBuilder/icon_ellipsisVertical.png"> (3-dot icon) in the upper-left corner, just to the right of the menu bar.
+1. Open the bot, and click <img style="width:25px" src="img/ConvoBuilder/icon_ellipsisVertical.png" alt="Three-dot icon"> (3-dot icon) in the upper-left corner, just to the right of the menu bar.
```

Comment:
Interesting case. What do you think about rephrasing this to `…and click the 3-dot icon (<img src="img/ConvoBuilder/icon_ellipsisVertical.png" alt="">)…`? In this case, an empty “alt” text would be okay.